public/app/plugins/datasource/loki/partials/config.html

```diff
@@ -1,2 +1,16 @@
 <datasource-http-settings current="ctrl.current" no-direct-access="true">
-</datasource-http-settings>
\ No newline at end of file
+</datasource-http-settings>
+
+<div class="gf-form-group">
+  <div class="gf-form-inline">
+    <div class="gf-form">
+      <span class="gf-form-label width-8">Line limit</span>
+      <input type="text" class="gf-form-input width-8" ng-model="ctrl.current.jsonData.queryLimit" spellcheck='false' placeholder="1000"></input>
```

Comment:
not sure query limit (property name) is the best here as it indicates a limit to number of queries. Maybe
* maxLines
* maxLineCount
* lineCountLimit 
* lineLimit

etc